src/polytopes/simplex.jl

```diff
@@ -37,3 +37,21 @@ nvertices(::Type{<:Simplex{K}}) where {K} = K + 1
 
 Base.isapprox(s₁::Simplex, s₂::Simplex; kwargs...) =
   all(isapprox(v₁, v₂; kwargs...) for (v₁, v₂) in zip(vertices(s₁), vertices(s₂)))
+
+"""
+    normal(splx::Simplex)
+
+Compute normal vector to simplex with euclidean norm of 1.
+"""
```

Comment:
```suggestion
```

Reply:
Why do you not want this docstring? E.g. it's not self-evident that the norm is 1.

Reply:
Fair enough. It's not normalized for `Ngon`. The `normal(::Simplex)` will always yield a normalized vector by mathematical construction, but I suppose we don't need to guarantee it in the API.